Makefile

```diff
@@ -8,10 +13,73 @@ DOCKER_IMAGE_NAME := node-feature-discovery
 
 VERSION := $(shell git describe --tags --dirty --always)
 
+ifeq ($(ARCH),amd64)
+	BASEIMAGE1 ?= golang:1.11-stretch
+	BASEIMAGE2 ?= debian:stretch-slim
+endif
+ifeq ($(ARCH),arm)
+	BASEIMAGE1 ?= arm32v7/golang:1.11-stretch
+	BASEIMAGE2 ?= arm32v7/debian:stretch-slim
+	QEMUARCH = arm
+endif
+ifeq ($(ARCH),arm64)
+	BASEIMAGE1 ?= arm64v8/golang:1.11-stretch
+	BASEIMAGE2 ?= arm64v8/debian:stretch-slim
+	QEMUARCH = aarch64
+endif
+ifeq ($(ARCH),ppc64le)
+	BASEIMAGE1 ?= ppc64le/golang:1.11-stretch
+	BASEIMAGE2 ?= ppc64le/debian:stretch-slim
+	QEMUARCH = ppc64le
+endif
+ifeq ($(ARCH),s390x)
+	BASEIMAGE1 ?= s390x/golang:1.11-stretch
+	BASEIMAGE2 ?= s390x/debian:stretch-slim
+	QEMUARCH = s390x
+endif
+
+check-docker:
+	@if [ -z $$(which docker) ]; then \
+          echo "Missing \`docker\` client which is required for development"; \
+          exit 2; \
+        fi
```

Comment:
I think this check should be removed. The build tool is supposed to be available, and, it might be something else than docker

Reply:
Done.